app/controllers/friends_controller.rb

```diff
@@ -0,0 +1,8 @@
+class FriendsController < ApplicationController
+
+  def create
+    @friend = Friend.create(user_id: current_user.id, friend_id: params[:friend][:friend_id])
```

Comment:
remember you can do user: current_user - you don't have to do user_id: current_user.id